scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: Command Injection

This function is vulnerable to command injection attacks. User input is directly concatenated into a shell command without proper sanitization[^1].

Recommendation: Avoid using `child_process.exec` with user input. If you must use it, ensure that user input is properly sanitized and validated before being used in command execution.

[^1]: CWE-78: Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection') - https://cwe.mitre.org/data/definitions/78.html
```suggestion
function executeCommand(userInput: string) {
    // Use child_process.execFile instead, which doesn't invoke a shell
    child_process.execFile('ls', [userInput], (error, stdout, stderr) => {
        if (error) {
            console.error(`exec error: ${error}`);
            return;
        }
        console.log(`stdout: ${stdout}`);
        console.error(`stderr: ${stderr}`);
    });
}
```